src/service/url-replacements-impl.js

```diff
@@ -74,6 +74,17 @@ function dateMethod(method) {
   return () => new Date()[method]();
 }
 
+/**
+ * Returns a function that executs method on the viewport.
```

Comment:
"executes"

I'd also add a comment describing why we're doing this. @erwinmombay FYI related to CC public property renaming.